### PR TITLE
Improvements/refactor

### PR DESCRIPTION
**Pull request by a7madM**
Changes in Dockerfile, docker-compose
- you don't need to specify dockerfile in docker-compose if you named it Dockerfile
- apt-get => apt, no more need to write apt-get in newer version of ubuntu
- App Volume, I think it should be relative to app, not absolute path, no need to write your name :D

in Source Code
- Golang style recommends to write a comment for public/exported functions/variables.
- if your variable/function doesn't be used outer its file, no need to make it public, just write it camelCase, not in this CamelCase.
- every file should have an empty end line

Good Luck, and thank you <3

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -3,4 +3,4 @@ TWITTER_API_SECRET=
 TWITTER_ACCESS_TOKEN_KEY=
 TWITTER_ACCESS_TOKEN_SECRET=
 
-PIC_STORAGE_PATH=/go/src/github.com/ahmedash95/shareAsPic/pics/
\ No newline at end of file
+PIC_STORAGE_PATH=./pics/
```
Comment (ahmedash95) [inline .env.example:6]: we need the absolute path to use it for photo upload, I can use os.Getwd() but I prefer to have the full path, can be fixed to /app/pics

Comment (a7madM) [reply .env.example:6]: what is the problem in this ?
it will upload to dir pics, in your WORKDIR in Docker Image

Comment (ahmedash95) [reply .env.example:6]: not needed. ignore my comment :D

**Comment by ahmedash95** [comment]
Thanks ❤️

**Pull request merged**